Sources/Fuzzilli/Core/Events.swift

```diff
@@ -64,6 +64,9 @@ public class Events {
     /// Signals that a new interesting program has been found, after the program has been minimized.
     public let InterestingProgramFound = Event<(program: Program, isImported: Bool, newTypeCollectionRun: Bool)>()
 
+    /// Signals that the REPRL instance encountered an error with the attached log of scripts.
+    public let REPRLFail = Event<[String]>()
```

Comment:
How about making this a generic `DiagnosticsEvent<name: String, content: String>` with the name/type used as filename and the (opaque) content string used as file content? Then this could be reusable for other things we might want to debug in the future. For example, we could more or less immediately reuse it to also output the entire "REPRL trace" for crashes in case they don't reproduce due to some "state leakage" from a previous program (but probably we'd need to give `config.diagnostics` some kind of verbosity level to not fill the disk with invalid programs, but that's for another time :) )
WDYT?

Reply:
Sounds good! :)